app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java

```diff
@@ -38,12 +38,11 @@ public DroidKaigiClient(DroidKaigiService droidKaigiService, GithubService githu
         this.googleFormService = googleFormService;
     }
 
-    public Single<List<Session>> getSessions(@NonNull String languageId) {
-        switch (languageId) {
-            case LocaleUtil.LANG_JA:
-                return droidKaigiService.getSessionsJa();
-            default:
-                return droidKaigiService.getSessionsEn();
+    public Single<List<Session>> getSessions(@NonNull Locale locale) {
+        if (locale == Locale.JAPAN) {
```

Comment:
In my understanding, `Locale.JAPAN` represents country.
`Locale.JAPANESE` seems better 🤔 

- Locale.JAPAN => ja_JP
- Locale.JAPANESE => ja